drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1324,6 +1324,14 @@ void TextureStorage::render_target_free(RID p_rid) {
 	}
 	render_target_owner.free(p_rid);
 }
+void TextureStorage::render_target_set_force_high_precision(RID p_render_target, bool p_force_high_precision) {
+	RenderTarget *rt = render_target_owner.get_or_null(p_render_target);
+	ERR_FAIL_COND(!rt);
+	if (rt->force_high_precision != p_force_high_precision) {
+		rt->force_high_precision = p_force_high_precision;
+		_update_render_target(rt);
+	}
```

Comment:
```suggestion
	rt->force_high_precision = p_force_high_precision;
```
There is no point in updating the render target if ``rt->force_high_precision`` is ignored anyway.

You may want to consider implementing ``force_high_precision`` and ``keep_linear`` in the GLES3 backend. It should not be much different than in the RendererRD backend

Reply:
``!spec->ok`` means that the engine is trying to draw with a shader that failed to compile. If you go to the top of your errors you should have a more complete error message that explains why the shader is failing to compile.

Reply:
Thanks i had indeed an error in scene shader. I'm unable to make my test project works in GLES3. It looks like something is clamping data between 0.0 and 1.0 outside of the linear call i found. I also noticed several issue in GLES3, for example my flag `keep_linear` when updated on my viewport is applied to all viewports (including the one used by editor).
Overall implementation of scene shaders in GLES3  is really tricky to  navigate in.
I'll get back to that later and create a simpler test project to share.

Reply:
By reading https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glTexImage2D.xhtml i understood that i had to use a custom format.
Example is now working in GLES3.
Not sure if i forgot some case in the `scene.glsl` shader but try a bunch of setup and everything works as expected.